### PR TITLE
introduce Phabricator support

### PR DESCRIPTION
**Pull request by okias**
#### Summary
Introduce Phabricator support.

#### Ticket Link

https://github.com/mattermost/mattermost-mobile/issues/6597

#### Checklist
- [ ] Added or updated unit tests (required for all new features)
- [x] Has UI changes
- [x] Includes text changes and localization file updates
- [ ] Have tested against the 5 core themes to ensure consistency between them.

#### Device Information
This PR was tested on: 
 - Pixel 5 (GrapheneOS, Android 13)
 - Pixel 7 (Stock, Android 13)
 - Pixel 7 Pro (Stock, Android 13)

#### Release Note

```release-note

```

### CODE REVIEW

**Comment by larkox** [comment]
@okias Feel free to ping me when the PR is out of the draft stage and ready to review :smile:

File: .circleci/config.yml [hunk 1]
```diff
@@ -566,9 +566,6 @@ workflows:
           context: mattermost-mobile-android-pr
           requires:
             - test
-          filters:
-            branches:
-              only: /^(build|android)-pr-.*/
```
Comment (enahum) [inline .circleci/config.yml:571]: Please revert this change

**Review by enahum — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (okias) [reply .circleci/config.yml:571]: Should I rename the branch and resend the MR to generate APK for testing?

Comment (enahum) [reply .circleci/config.yml:571]: The PR for testing it needs to be done by someone from the org.. ping me when you want me to trigger the build

Comment (okias) [reply .circleci/config.yml]: @enahum rebased, could you trigger APK build?

Comment (okias) [reply .circleci/config.yml]: @enahum one more ping, sadly the Phabricator Icon asset was missing from the PR :disappointed:
